packages/client-core/src/common/services/ProjectService.ts

```diff
@@ -106,10 +106,13 @@ export const ProjectServiceReceptor = (action) => {
 //Service
 export const ProjectService = {
   fetchProjects: async () => {
-    const projects = await API.instance.client.service(projectPath).find({ paginate: false, query: { allowed: true } })
-    dispatchAction(ProjectAction.projectsFetched({ projectResult: projects.data }))
-    for (let error of projects.errors) {
-      NotificationService.dispatchNotify(error.message || JSON.stringify(error), { variant: 'error' })
+    try {
+      const projects = (await API.instance.client
+        .service(projectPath)
+        .find({ paginate: false, query: { allowed: true } })) as ProjectType[]
```

Comment:
`paginate: false` is not supported from client. If you want to retrieve all records then you can have a look of following:

- https://feathersjs.com/api/databases/common.html#pagination (Check the Note: in the end of this section)
- https://github.com/feathersjs/feathers/issues/382#issuecomment-288125825

Also aur service is always returning array.